ui/src/app/components/prepare-migration/prepare-migration.component.html

```diff
@@ -115,6 +115,19 @@ <h3>Target Details</h3>
         <mat-progress-bar mode="determinate" [value]="dataMigrationProgress"></mat-progress-bar>
         <span> {{this.dataProgressMessage}}</span>
     </div>
+    <div *ngIf="isResourceGenerated">
+        <h3> Generated Resources:</h3>
+        <li>Spanner database: <a [href]="resourcesGenerated.DatabaseUrl">{{resourcesGenerated.DatabaseName}}</a></li>
+        <li>GCS bucket: <a [href]="resourcesGenerated.BucketUrl">{{resourcesGenerated.BucketName}}</a></li>
```

Comment:
nit: BucketName maps to BucketUrl ?
Same for all other fields. If possible use the same names ?

Reply:
That's because in UI it will show name of these elements with links which can be used to go the asset